src/SSHDebugPS/Microsoft.SSHDebugPS.pkgdef

```diff
@@ -2,14 +2,24 @@
 "CLSID"="{1326FB0D-EA51-4F90-BEDF-5948588B0FE1}"
 "Name"="SSH"
 
+[$RootKey$\AD7Metrics\PortSupplier\{A2BBC114-47E4-473F-A49C-69EE89711243}]
+"CLSID"="{18F976B7-D66D-4C36-9CAE-A9217E0E3DF4}"
+"Name"="Docker"
```

Comment:
>Docker [](start = 8, length = 6)

I wonder if we should call this `Docker (Linux)` to make it clear that it doesn't support Windows scenarios?

Reply:
I was thinking about how to name it to specify that we support only Docker Linux containers for now.

Docker (Linux) might lead users to believe Docker On Linux?

Reply:
Could go with "Docker (Linux Container)" or something, to indicate that you can attach to any linux-based container, regardless of the host OS.